backend/internal/testutils/testutils.go

```diff
@@ -17,6 +17,10 @@ func MakeHttpRequest(method string, url string, bodyString string, expectedStatu
 	req, err := http.NewRequest(method, url, bodyReader)
 	require.NoError(t, err)
 
+	if method == "POST" || method == "PUT" {
```

Comment:
We're using `PATCH`es :) Might as well leave the method == "PUT" expression in here too though, just add `|| method == "PATCH"` to the end, just in case we ever decide to add any PUT routes.